misc/dist/macos_template.app/Contents/Info.plist

```diff
@@ -24,6 +24,22 @@
 	<string>$signature</string>
 	<key>CFBundleVersion</key>
 	<string>$version</string>
+	<key>DTCompiler</key>
+	<string>com.apple.compilers.llvm.clang.1_0</string>
```

Comment:
I've narrowed it down a little https://github.com/godotengine/godot/issues/74154#issuecomment-1450162767 and not all of these (like the compiler key) seem required.

Reply:
Not required for now, but this can change at any moment. Since Xcode always set it, it's probably better to keep it as well.